build/config/linux/pkg-config.py

```diff
@@ -203,7 +205,11 @@ def main():
         sys.stdout.write(dridriverdir.strip())
         return
 
-    cmd = [options.pkg_config, "--cflags", "--libs"] + args
+    if options.static:
+        cmd = [options.pkg_config, "--cflags", "--libs", "--static"] + args
+    else:
+        cmd = [options.pkg_config, "--cflags", "--libs"] + args
```

Comment:
```suggestion
    cmd = [options.pkg_config, "--cflags", "--libs"]

    if options.static:
        cmd.append("--static")
        
    cmd.extend(args)
```

seems a bit less repeating and may extend more easily later if we add more flags.